released/packages/coq-sail/coq-sail.0.16/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+name: "coq-sail"
+version: "0.16"
```

Comment:
```suggestion
```
This is already given by the repository `opam` file path.

Reply:
Oops, an artifact of starting from the file used for pinning...